cassandra/project.clj

```diff
@@ -5,7 +5,7 @@
             :url "http://www.eclipse.org/legal/epl-v10.html"}
   :dependencies [[org.clojure/clojure "1.10.1"]
                  [org.clojure/java.jmx "0.3.1"]
-                 [jepsen "0.1.15"]
+                 [jepsen "0.1.18"]
```

Comment:
Just a question. Any reason to use `18` ? `19` seems the latest now.

Reply:
Thank you for your question.
That's right. But I couldn't build the current tests with `0.1.19`.
I'd like to support that in another PR.